cmd/proxy/actions/app.go

```diff
@@ -129,7 +129,8 @@ func App(conf *config.Config) (*buffalo.App, error) {
 
 	// Having the hook set means we want to use it
 	if vHook := conf.ValidatorHook; vHook != "" {
-		app.Use(mw.LogEntryMiddleware(mw.NewValidationMiddleware, lggr, vHook))
+		app.Use(mw.LogEntryMiddleware(lggr))
```

Comment:
Let's move this outside to be the top level middleware. AKA the first thing that an http request hits, so that all middlewares and http requests have access to the log.Entry.

Reply:
@marwan-at-work should be addressed now right?

Reply:
actually i see this is already done...gonna give this one more review and merge if peeps are happy with it